doc/api/net.md

```diff
@@ -870,6 +870,11 @@ behavior.
 <!-- YAML
 added: v0.1.90
 changes:
+  - version: REPLACEME
+    pr-url: REPLACEME
```

Comment:
```suggestion
    pr-url: https://github.com/nodejs/node/pull/46587
```

Reply:
Yup, I'm waiting for this to be created. Anyway, I'll move this changes to another PR.